test/v1/controllers/SubmitReturnControllerSpec.scala

```diff
@@ -171,6 +166,28 @@ class SubmitReturnControllerSpec
       }
     }
 
+//    "a valid request is supplied but NRS is failed" should {
+//      "return the INTERNAL_SERVER_ERROR" in new Test {
+//
+//        MockSubmitReturnRequestParser
+//          .parse(submitRequestRawData)
+//          .returns(Right(submitReturnRequest))
+//
+//        MockNrsService
+//          .submitNrs(submitReturnRequest, date)
+//          .returns(Future.successful(Left(ErrorWrapper(correlationId, DownstreamError, None))))
+//
+//        private val result: Future[Result] = controller.submitReturn(vrn)(fakePostRequest(submitRequestBodyJson))
+//
+//        status(result) shouldBe INTERNAL_SERVER_ERROR
+//        contentAsJson(result) shouldBe Json.toJson(DownstreamError)
+//
+//        val auditResponse: AuditResponse = AuditResponse(INTERNAL_SERVER_ERROR, Some(Seq(AuditError("INTERNAL_SERVER_ERROR"))), None)
+//        MockedAuditService.verifyAuditEvent(AuditEvents.auditSubmit(correlationId,
+//          UserDetails("Individual", None, "N/A"), auditResponse)).once
+//      }
+//    }
+
```

Comment:
If these tests aren't needed any more can we remove them?

Reply:
The requirement is that even if NRS fails, our journey succeeds, so not sure if the tests below are correct